crates/codegen/parser_runtime/src/templates/language.tera

```diff
@@ -0,0 +1,236 @@
+#[cfg(feature = "slang_napi_interfaces")]
+use { napi::bindgen_prelude::*, napi_derive::napi };
+
+use semver::Version;
+
+use super::{
```

Comment:
Using templates (instead of the old `syntax_shared` crate) enable us to to move some of this code to the output crate directly. For example `Error` and its extensions. This simplifies the template, and gives us intellisense there. Did this change?

Definetely not a blocker to this PR, and I can take a stab at it later.

Reply:
"move some of this code to the output crate directly" - not sure what you mean here. It is in the runtime crate that is copied to the output crate.

I much prefer to have the output crate be just that - an output crate. The runtime crate being next to the parser generator crate is a good model IMO.

Reply:
Sounds good. Thanks!